test/sampleData/Mcp4725.py

```diff
@@ -100,3 +107,28 @@ def set_eeprom(self, data):
             self.REGISTER_EEPROM,
             data
         )
+
+    def get_digitalout(self):
+        """
+        Only allows you to send fully on or off
+
+        """
+        # Read register data
+        # '#/registers/EEPROM' > 'EEPROM'
+        val = self.get_eeprom()
+        val = _swap_endian(val)
+        # Mask register value
+        val = val & 0b0000111111111111
+        return val
+
+    def set_digitalout(self, data):
```

Comment:
I'm not sure. Stylistically it can be better for trivial methods, but these can end up being non-trivial.

Reply:
Yea, that is true. I'd recommend that if we support direct access to changing the registers, it should be with the builtins and more complicated access (values or psudoyaml) with getters and setters.

Reply:
Would it make sense to make the code styles inconsistent?

Reply:
@chrisfrederickson should I refactor the template to use property syntax at the register-level? If so, I'll probably do that in a separate pull request.

Reply:
@Fleker I think it is fine keeping it as is for the time being. Eventually it may make sense to add an option to the Python generator to support both styles.